packages/forms/src/Components/Concerns/CanBeValidated.php

```diff
@@ -126,12 +126,14 @@ public function same(string | Closure $statePath, bool $isStatePathAbsolute = fa
         return $this->fieldComparisonRule('same', $statePath, $isStatePathAbsolute);
     }
 
-    public function unique(string | Closure | null $table = null, string | Closure | null $column = null, Model | Closure $ignorable = null, ?Closure $callback = null): static
+    public function unique(string | Closure | null $table = null, string | Closure | null $column = null, Model | Closure $ignorable = null, ?Closure $callback = null, bool $ignoreCurrent = false): static
     {
-        $this->rule(function (Field $component, ?string $model) use ($callback, $column, $ignorable, $table) {
+        $this->rule(function (Field $component, ?string $model) use ($callback, $column, $ignorable, $table, $ignoreCurrent) {
```

Comment:
I feel like it should be `ignoreRecord` or `ignoreCurrentRecord`, `ignoreCurrent` doesn't describe what exactly we're ignoring

Reply:
`ignoreRecord` is fine, will update.